src/components/detailsPage/pages/ontologies/OntologiesDetailsPage.tsx

```diff
@@ -24,9 +26,13 @@ const OntologiesDetailsPage: FC = () => {
   const [ontology, setOntology] = useState<Ontology>();
 
   useEffect(() => {
+    // TODO: Isn't it already available? Is it necessary to fetch the ontology? The view is probably opened from a
+    //  search view where ontologies already have been fetched
```

Comment:
In theory it should be possible to send someone the link to the detail page of an ontology. So you dont necessarily come from the Onologies page. But the direct link opening doesnt work right now

Reply:
Thank you for the answere. I still think there might be a soultion to detect if we come from a main page and have the data already fetched or we opened the detail page directly entering a link. (**hint**: using some kind of local storage solution: contex or may be redux.) My focus is currently on the main pages I do not have the capacity to think it thorugh. I'll come back to this issue later. I wrote the comment to remind myself. But I can also remove it.